board-viewer.html

```diff
@@ -463,6 +464,42 @@
     }
   },
 
+  _onMultiSelectEntering: function(event) {
+    if (event.keyCode == 91 || event.keyCode == 93 || event.keyCode == 224) {
```

Comment:
This will only work on the MAC. You need to add back in the event.ctrlKey check as well to work on Linux and Windows, similar to what I called out in the previous review comments.

Rather than performing the functional logic in the key handler, the key handler should be limited to only setting the `multiSelectActive` boolean property. The property observer for that would then contain the functional logic.

When the property is set to true, **only** disable the shadow; it shouldn't clear out any existing filters as the user may be adding items to the current selected state.

When the property is set to false, turn the shadow on.

The "selected" filters would only be cleared out when the user selects an item **without** the multi-select active.